Plugins/FormatSwift/Plugin.swift

```diff
@@ -96,6 +115,32 @@ struct AirbnbSwiftFormatPlugin: CommandPlugin {
 
 }
 
+#if canImport(XcodeProjectPlugin)
+extension AirbnbSwiftFormatPlugin: XcodeCommandPlugin {
+
+  func performCommand(context: XcodePluginContext, arguments: [String]) throws {
+    var argumentExtractor = ArgumentExtractor(arguments)
+
+    // When ran from Xcode, the plugin command is invoked with `--target` arguments,
+    // specifying the targets selected in the plugin dialog.
+    //  - Unlike SPM targets which are just directories, Xcode targets are
+    //    an arbitrary collection of paths.
+    let inputTargetNames = Set(argumentExtractor.extractOption(named: "target"))
+    let inputPaths = context.xcodeProject.targets.lazy
```

Comment:
Why make this `lazy` if we're just converting it to an array below?

Reply:
I'd expect `lazy` here to improve performance a bit for large targets with lots of files by reducing the total number of loop iterations (e.g. not having to do two separate loop passes for the `map` and then `filter`). Could be mistaken though.

Reply:
Got it, makes sense, so we get better data locality since each of the operations for a single element occur in sequence?

Reply:
Yeah I think so

Reply:
Maybe it doesn't really matter / is a micro-optimization, I didn't instrument it of course. I just tend to lean for `lazy` in cases where a functional chain on a collection has a bunch of different steps like this.